.github/ISSUE_TEMPLATE/marketing-request.yml

```diff
@@ -0,0 +1,11 @@
+name: Contributor Comms Request
+description: Get support on sending a message out to our community through coordinated outreach on email, Slack, social media, and other public channels.
+title: 'REQUEST: New communication about <topic>'
+labels: area/contributor-comms, sig/contributor-experience
+body:
+- type: input
```

Comment:
This should be a textarea, they'll likely need to expand on whatever request they're submitting 👍